config-rules.tf

```diff
@@ -5,7 +5,8 @@ module "aws_config_rules_us_east_1" {
   source_recorder               = module.aws_config_recorder_us_east_1.aws_config_configuration_recorder_id
   required_tags_enabled         = true
   required_tags = {
-    tag1Key = "Environment" # Yes, the actual required format is tag#Key and the required key
```

Comment:
@jason-drouhard - Why did we remove the "Environment" tag requirement?

Reply:
I removed the "Environment" tag because we tag resources at a provider level... and the environment is included in that. With it being at the provider level, it tags every possible resource with the tag you specify. I guess in my mind it didn't seem necessary to make it a required tag at that point. If you disagree, I will add it back.

Reply:
I would keep it in there. Both the environment tag, and this backup tag monitor, is to catch when something isn't being done correctly. This includes when it is being done in the console or similar.

Reply:
@StratusChris I will make sure all these changes are implemented today.